app/Models/UserPref.php

```diff
@@ -99,9 +99,6 @@ protected function setKeysForSaveQuery($query)
     {
         /** @var array */
         $keys = $this->getKeyName();
-        if (! is_array($keys)) {
```

Comment:
> Call to function is_array() with array will always evaluate to true.

The key is an array, not sure what it's suppose to check?

`protected $primaryKey = ['user_id', 'pref'];`